backpack-common/src/main/java/net/skyscanner/backpack/calendar2/data/CalendarStateMachine.kt

```diff
@@ -142,7 +144,7 @@ internal fun CalendarState.dispatchSetSelection(selection: CalendarSelection): C
     )
 }
 
-internal fun CalendarState.dispatchClick(data: CalendarCell.Header): CalendarState {
```

Comment:
exposed the above internal external functions to be reused by `BpkCalendarController`